Refactor(frontend): Move frontend folders to root & updated npm

Maasdamind opened the pull request:
…e.json

tjomson commented:
Should src in the root be moved to app/src?
![billede](https://user-images.githubusercontent.com/23435481/110126915-3cfa5480-7dc5-11eb-8e6d-84158ecac01c.png)

Maasdamind commented:
@tjomson @hojelse  So i've moved the stuff around now and removed `app.` if we were to put `src` into `app`, we would have to specify the location in the frontend commands we have. Atm it will now find our `main.ts` in `src` when doing `npm run serve`.  Since the backend will be contained in one folder, i think it's safe enough to spread the other folders out in the root. Let me know what you think.

tjomson reviewed (approved):
Looks like a good structure 👍
